containers/Containerfile

```diff
@@ -0,0 +1,87 @@
+FROM registry.fedoraproject.org/fedora-toolbox:34
+
+MAINTAINER "Candlepin Team" <candlepin@redhat.com>
+
+ENV SMDEV_CONTAINER_OFF='True'
+
+RUN yum install --disablerepo=updates -y \
+    bind-utils \
+    cmake \
+    dbus-daemon \
+    dbus-glib \
+    dbus-glib-devel \
```

Comment:
I believe that this packages was required, because rhsm-icon communicate using D-Bus, but we don't need it anymore, because other compiled tools do not use D-Bus API.